src/platform/plugins/shared/usage_collection/common/constants.ts

```diff
@@ -10,3 +10,4 @@
 export const KIBANA_STATS_TYPE = 'kibana_stats';
 export const DEFAULT_MAXIMUM_WAIT_TIME_FOR_ALL_COLLECTORS_IN_S = 1;
 export const MAIN_APP_DEFAULT_VIEW_ID = 'main';
+export const USAGE_COUNTERS_BACKOFF_DELAY_IN_MS = 500;
```

Comment:
Exports the backoff delay as a constant so that it doesn't get "lost" as a magic number in the usage counters service.